ts/features/idpay/code/onboarding/screens/IDPayCodeDisplayScreen.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Banner,
+  Body,
+  ButtonSolid,
+  IOColors,
+  IOSpacingScale,
+  Label,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+
+import { useNavigation } from "@react-navigation/native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { NewH3 } from "../../../../../components/core/typography/NewH3";
+import TopScreenComponent from "../../../../../components/screens/TopScreenComponent";
+import I18n from "../../../../../i18n";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+
+const CodeDigitDisplayBox = ({ digit }: { digit: string }) => (
+  <View style={styles.codeDigit}>
+    <NewH3>{digit}</NewH3>
+  </View>
+);
+const CodeDisplayComponent = ({ code }: { code: string }) => (
+  <View style={styles.codeDisplay}>
+    {code.split("").map((digit, index) => (
+      <CodeDigitDisplayBox key={index} digit={digit} />
+    ))}
+  </View>
+);
+
+export const IDPayCodeDisplayScreen = () => {
+  const code = "12345"; // get from store
+  const viewRef = React.useRef(null);
+  const navigation = useNavigation(); // TODO:: delete
+  return (
+    <SafeAreaView style={styles.mainContainer}>
+      <TopScreenComponent contextualHelp={emptyContextualHelp}>
+        <NewH3>{I18n.t("idpay.code.onboarding.header")}</NewH3>
```

Comment:
```suggestion
        <H2>{I18n.t("idpay.code.onboarding.header")}</H2>
```

Reply:
isn't this H2 not responsive to the DS switch changes?

Reply:
I don't know, maybe `NewH2`? `NewH3` is not the correct typography.

Reply:
AFAIK there is no `NewH2` in the current DS version, I guess we should make it 😞

Reply:
my bad, apparently `H2` behaves like you'd expect `NewH2` to, so I applied It.
Also, IIRC there's an update about namings in the works to avoid this, correct?